boards/arm/ip_k66f/Kconfig.board

```diff
@@ -5,3 +5,13 @@ config BOARD_IP_K66F
 	bool "Segger IP-K66F"
 	depends on SOC_SERIES_KINETIS_K6X
 	select SOC_PART_NUMBER_MK66FN2M0VMD18
+
+config BOARD_INIT_PRIORITY
+	int
+	default 47
+	depends on BOARD_IP_K66F
+	depends on GPIO
```

Comment:
Having this depend on GPIO will probably not do what you want here. If GPIO is disabled, `CONFIG_BOARD_INIT_PRIORITY` will be undefined yet `board.c` will still be compiled?

Reply:
You are correct - the dependency on GPIO shall be removed.